src/main/java/software/sigma/internship/mapper/AnswerMapper.java

```diff
@@ -18,7 +18,7 @@ public Answer toEntity(AnswerDto dto) {
     public AnswerDto toDto(Answer entity) {
         AnswerDto dto = new AnswerDto();
         dto.setId(entity.getId());
-        dto.setQuestion(entity.getQuestion());
+        dto.setQuestion(null);
```

Comment:
I think you don't need this row. It is already null.

Reply:
Agree. Also removed similar line from QuestionDto